src/main/java/fr/cnes/sonar/report/exporters/docx/DocXExporter.java

```diff
@@ -115,6 +122,27 @@ public File export(final Object data, final String path, final String filename)
             XWPFDocument document = new XWPFDocument(opcPackage)
         ) {
 
+            // Search the number of security hotspots
+            List<Issue> allIssues = report.getIssues();
+            int counter = 0;
+            for(Issue issue : allIssues){
+                if (issue.getType().equals(StringManager.HOTSPOT_TYPE)){
+                    counter++;
+                }
+            }
+            // Put this number in the corresponding facet "severities"
+            report.setIssues(allIssues);
+            List<Facet> facets = report.getFacets();
+            for(Facet facet : facets){
+                if(facet.getProperty().equals(SEVERITIES)){
+                    List<Value> values = facet.getValues();
+                    for(Value value : values){
+                        if(value.getVal().equals(StringManager.HOTSPOT_SEVERITY)){
+                            value.setCount(value.getCount() + counter);
+                        }
+                    }
+                }
+            }
```

Comment:
I think it is not the best place to edit the model regarding the chosen architecture, move this code in data adaptor or model generation.